### PR TITLE
feat(py-sdk): add manifest to package [SBK-239]

### PR DESCRIPTION
**Pull request by fubuloubu**
fixes: #20

### CODE REVIEW

File: sdk/py/apepay/manifest.json [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../.build/__local__.json
```
Comment (antazoey) [inline sdk/py/apepay/manifest.json:1]: i am confused by this file, it is not really JSON?

File: sdk/py/apepay/__init__.py [hunk 2]
```diff
@@ -54,26 +55,47 @@ def validate(self, creator, token, amount_per_second, reason) -> bool:
...
+    _local_contracts: Dict[str, ContractType] = {}
 
     @validator("address", pre=True)
     def normalize_address(cls, value: Any) -> AddressType:
         return cls.conversion_manager.convert(value, AddressType)
 
-    @cached_property
-    def _local_contracts(self) -> Dict[str, ContractType]:
+    def __post_init__(self):
+        # 0. If pre-loaded, default that type
```
Comment (antazoey) [inline sdk/py/apepay/__init__.py:65]:
```suggestion
        # 0. If pre-loaded, default to that type
```

**Review by antazoey — APPROVED** [review]

File: sdk/py/apepay/manifest.json (hunk 1 above)
Comment (fubuloubu) [reply sdk/py/apepay/manifest.json:1]: This is a soft link back to the build file, so that poetry will bundle it

Comment (fubuloubu) [reply sdk/py/apepay/manifest.json:1]: when you install it, it ends up in `site-packages/.build/`

Comment (fubuloubu) [reply sdk/py/apepay/manifest.json:1]: I suppose a better solution is to just cp it into the directory when it gets built :thinking:

**Review by antazoey — APPROVED** [review]

**Comment by fubuloubu** [comment]
> need to debug how to get it installed correctly, still not doing it right lol

Okay figured this out, file just wasn't available lol

File: sdk/py/apepay/manifest.json (hunk 1 above)
Comment (fubuloubu) [reply sdk/py/apepay/manifest.json:1]: It fails now if you don't build first, so I like this approach as it's more fail-safe

**Comment by fubuloubu** [comment]
Added link to #20 because this resolves that issue

**Review by antazoey — APPROVED** [review]
seems good to me!!

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -13,24 +13,39 @@ concurrency:
...
+  py-test:
+    needs: build
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: foundry-rs/foundry-toolchain@v1
+      - uses: ApeWorX/github-action@v2
+      # Ape build artifacts used for JS testing
```
[the remark this answers is not in the document]
Comment (fubuloubu) [reply .github/workflows/test.yaml]: Just a bad comment, sorry

**Pull request merged**